### PR TITLE
Sprint12 05 a45638 sys second review

### PR DESCRIPTION
**Pull request by ekargard**

### CODE REVIEW

File: cookbooks/sys/README.rdoc [hunk 1]
```diff
@@ -33,10 +33,8 @@ There are no known limitations.
 
 === System Consistency (re-convergence)
 
-  To enforce a consistent system state, one can use this LWRP to schedule
-  a cron job to periodically run a recipe.  Currently the period is hard-coded
-  to every 15 minutes -- with a random starting offset (to distribute runs
-  being performed by multiple systems).
```
Comment (Neve) [inline cookbooks/sys/README.rdoc:39]: maybe we should leave info about random starting offset?

Comment (lopaka) [inline cookbooks/sys/README.rdoc:39]: I agree, we should keep the offset info.

Comment (douglaswth) [inline cookbooks/sys/README.rdoc:39]: I'm working on it!

Comment (douglaswth) [inline cookbooks/sys/README.rdoc:39]: Actually that offset is not actually used by this, it is on the haproxy/db/block_device stuff.

**Pull request merged**